operators/utilities.py

```diff
@@ -7,7 +7,7 @@
 
 from .. import global_data
 from ..declarations import GizmoGroups, WorkSpaceTools
-from ..class_defines import SlvsGenericEntity
+from ..class_defines import SlvsGenericEntity, point
```

Comment:
now that we changed the `if` statement, the `point` import isn't useful :)

Reply:
done